ssiva13/Questioner
# Ft admin create meetup 162977915

**What Does This PR Do?**
This PR contains the UI template for the create meetup feature.
**Description of Feature/Task**
Create create meetup feature page

**_How to Manually test this feature?_**

- Clone the repository using 
- cd into project folder
- Open create.html using a browser


> Pivotal tracker story ID _#162977915_

The page(as it should be)

![screenshot_2019-01-06 questioner 6](https://user-images.githubusercontent.com/20058478/50740503-70022700-1200-11e9-91a7-cef54eaec1c5.png)
